cumulus/zombienet/zombienet-sdk/tests/zombie_ci/full_node_catching_up.rs

```diff
@@ -0,0 +1,132 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::anyhow;
+use tokio::time::Duration;
+
+use crate::utils::BEST_BLOCK_METRIC;
+use cumulus_zombienet_sdk_helpers::assert_para_throughput;
+
+use polkadot_primitives::Id as ParaId;
+use subxt::{OnlineClient, PolkadotConfig};
+use zombienet_orchestrator::network::node::LogLineCountOptions;
+use zombienet_sdk::{LocalFileSystem, Network, NetworkConfigBuilder};
+
+const PARA_ID: u32 = 2000;
+
+// This tests makes sure that parachain full nodes are synchronizing with the validator
+// and report expected block height.
+#[tokio::test(flavor = "multi_thread")]
+async fn full_node_catching_up() -> Result<(), anyhow::Error> {
+	let _ = env_logger::try_init_from_env(
+		env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "info"),
+	);
+
+	log::info!("Spawning network");
+	let network = initialize_network().await?;
+
+	let relay_alice = network.get_node("alice")?;
+	let relay_client: OnlineClient<PolkadotConfig> = relay_alice.wait_client().await?;
+
+	log::info!("Ensuring parachain is registered");
+	assert_para_throughput(
+		&relay_client,
+		20,
+		[(ParaId::from(PARA_ID), 2..40)].into_iter().collect(),
```

Comment:
dq: How was the 2..40 interval determined? It looks like most of the zombienet tests fail in a flaky manner because of a similar range check

Reply:
I adjusted this value empirically by analyzing the grafana loki logs for failed and successful tests in CI.
The parachain blocks are starting to be backed up by relay chain in a different timeslots depending on the CI run.
I cannot tell what is the root cause of that.
Thus the expected range  2..40 within 20 relay chain blocks is pretty wide. 
In the original zndsl file it was:
```
alice: parachain 2000 is registered within 225 seconds
```
which seems pretty wide too, we were waiting up to 37 blocks (225s, where 1 block is 6s).